{
  "id": "MAL-2024-5289",
  "summary": "Malicious code in jxzspwqwup (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:36:38Z",
  "published": "2024-06-25T13:36:38Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:36:38Z",
        "import_time": "2024-06-28T02:49:17.193958016Z",
        "versions": [
          "0.0.1"
        ],
        "id": "RLMA-2024-04071",
        "source": "reversing-labs",
        "sha256": "0fe2aa8ceeefc9578345ac05afd73ae3cf6e1887c9700e0b88edef30a7ede8db"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "jxzspwqwup",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/jxzspwqwup"
      },
      "versions": [
        "0.0.1"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/jxzspwqwup/MAL-2024-5289.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}